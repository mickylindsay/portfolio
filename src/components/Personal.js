import React from 'react';

const Personal = props => {
  return (
    <h1>abc</h1>
  )
}
export default Personal;
