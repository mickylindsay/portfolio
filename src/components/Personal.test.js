import React from 'react';
import {shallow} from 'enzyme';
import Personal from './Personal';

describe('Personal component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Personal/>);
    expect(wrapper).toMatchSnapshot();
  });
});
