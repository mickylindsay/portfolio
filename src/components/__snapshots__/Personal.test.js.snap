// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Personal component matches snapshot 1`] = `
<h1>
  abc
</h1>
`;
